### PR TITLE
tests: do not use external snaps in spread

### PR DESCRIPTION
**Pull request by fgimenez**

### CODE REVIEW

File: tests/lib/snaps/test-snapd-python-webserver/meta/snap.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: test-snapd-python-webserver
+version: 16.04-3
+summary: Python based example webserver
+description: |
+ Show random XKCD comic via a build-in webserver
+ This is meant as a fun example for a snappy package.
+apps:
+ test-snapd-python-webserver:
+  command: ./bin/test-snapd-python-webserver
```
Comment (niemeyer) [inline tests/lib/snaps/test-snapd-python-webserver/meta/snap.yaml]: Can we please use 4-space indents on these files as well? A single space is too close for quickly skimming through.

**Comment by fgimenez** [comment]
@niemeyer thanks for the review! some snaps haven't reached the store yet, i'll retrigger when all is in place.

**Comment by mvo5** [comment]
:+1:

**Comment by mvo5** [comment]
A store unit test needs an update, maybe just a missing `git add` from the last commit?

**Pull request merged**